### PR TITLE
initial copy updates

### PR DESCRIPTION
**Pull request by deilann**
Intial copy. It's not perfect and not entirely optimized, but it's a lot better. See comments in file. Will need images, style, and real links. (But who doesn't need style?)

### CODE REVIEW

**Comment by paglias** [comment]
HabitRPG/habitrpg-shared#113 has to be merged together with this

**Comment by lefnire** [comment]
You're gonna wanna [use branching](http://git-scm.com/book/en/Git-Branching-Basic-Branching-and-Merging) to get the three main PRs you have floating around separated. Create a separate branch for each feature, PR them all against develop. Lets us know if you need help doing this. If it's something you're not familiar with and you'd rather me just get the show on the road, I can do some cherry-picking and close the PRs manaully

**Comment by colegleason** [comment]
If you are not familiar with git branching workflows, I highly recommend
learning them. It will make your life so much easier on large projects
(like this one).

I think I'm going to start working on a talk about them.

On Fri, Jan 24, 2014 at 7:44 PM, Tyler Renelle notifications@github.comwrote:

> You're gonna wanna use branchinghttp://git-scm.com/book/en/Git-Branching-Basic-Branching-and-Mergingto get the three main PRs you have floating around separated. Create a
> separate branch for each feature, PR them all against develop. Lets us know
> if you need help doing this. If it's something you're not familiar with and
> you'd rather me just get the show on the road, I can do some cherry-picking
> and close the PRs manaully
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/HabitRPG/habitrpg/pull/2547#issuecomment-33277753
> .

## 

Cole Gleason
Student, University of Illinois at Urbana-Champaign
Computer Science, Class of 2015
cg@colegleason.com | colegleason.com

**Comment by deilann** [comment]
I know how to do it. I just screwed up the first time (failed to actually
switch to the branch) and accidentally added extra commits the second time.
And besides, I'm a kinetic learner. :P Need to fix my errors in order to
learn from 'em.

File: views/static/front.jade [hunk 1]
```diff
@@ -10,15 +10,38 @@ block content
...
-        .container
-          h1
-            img(src='/bower_components/habitrpg-shared/img/logo/habitrpg_pixel.png', alt='HabitRPG')
-          p=env.t('synopsis')
-          a.btn.btn-primary.btn-small(ng-click='playButtonClick()')=env.t('playButton')
       .container
-        p(style='height:600;')
-          iframe(src='//player.vimeo.com/video/76557040', width='100%', height='539', frameborder='0', webkitallowfullscreen='', mozallowfullscreen='', allowfullscreen='')
+        //we need to use something that's not jumbotron for this, but still keep it centered
+        //could someone write something else to make it pretty?
```
Comment (lefnire) [inline views/static/front.jade]: I'll see what I can do!

Comment (deilann) [inline views/static/front.jade]: Awesome. The shared PR just updates the text for the feeding mount error,
fixes a typo, and has the strings for the copy. Do you still want me to
separate them?

**Comment by deilann** [comment]
Okay, this is now just the landing page copy.

**Pull request closed without merging**